Tests/WebSocketTests/ClientTests.swift

```diff
@@ -29,6 +29,7 @@ final class WebSocketClientTests: XCTestCase {
         }()
         try await WebSocketClient.connect(
             url: "wss://echo.websocket.org/",
+            configuration: .init(sniHostname: "echo.websocket.org"),
```

Comment:
Is this mandatory now? Isn't that a breaking change?

Reply:
No it is not. I added it in here to make sure the code path worked. 
I've added a second test which doesn't include it